gm4_washing_tanks/data/gm4_washing_tanks/functions/washing_recipes/wool.mcfunction

```diff
@@ -0,0 +1,6 @@
+#@s = water tank with one coloured wool in the first slot
+#run from wash
+
+data modify block ~ ~ ~ Items[0].id set value white_wool
```

Comment:
these aren't consistent with current liquid tank expansion packs. it should use the `replaceitem` command (tho this will be changed to `item modify` in 1.17). Any one that just sets the item to a new id should use `replaceitem`. The ones where you actually modify item data is good to use `data remove`

Reply:
I did this to preserve any names/enchantments/other stuff that might be on there, but seeing as other ones use replaceitem I should do that as well

Reply:
Oh that's probably a better idea; people may throw compressed items in here

Reply:
oh yes, that is also a problem, you would be able to wash 64 for the price of 1

Reply:
also shulker items